storage/backend/ceph/ceph.go

```diff
@@ -239,3 +239,38 @@ func (c *Driver) ListSnapshots(do storage.DriverOptions) ([]string, error) {
 
 	return names, nil
 }
+
+// ShowMapped describes all the volumes currently mapped on to the host. It
+// does not yield any mount points (yet.)
+func (c *Driver) ShowMapped() ([]*storage.Mount, error) {
+	out, err := exec.Command("rbd", "showmapped").Output()
+	if err != nil {
+		return nil, err
+	}
+
+	// FIXME maybe a new type here would be better than re-using this one.
+	mounts := []*storage.Mount{}
+	for i, line := range strings.Split(string(out), "\n") {
+		if i == 0 {
+			continue
+		}
+
+		parts := regexp.MustCompile(`\s+`).Split(line, -1)
```

Comment:
We should compile this regex only once and have it in the init. This can also be changed afterwards if you prefer that.